jwnl-addon/pom.xml

```diff
@@ -33,17 +33,28 @@
   <packaging>jar</packaging>
   <name>Apache OpenNLP JWNL Addon</name>
 
+  <properties>
+    <extjwnl.version>2.0.5</extjwnl.version>
+    <wn-data.version>1.2</wn-data.version>
+  </properties>
+
   <dependencies>
     <dependency>
       <groupId>org.apache.opennlp</groupId>
       <artifactId>opennlp-tools</artifactId>
     </dependency>
 
     <dependency>
-      <groupId>net.sf.jwordnet</groupId>
-      <artifactId>jwnl</artifactId>
-      <version>1.3.3</version>
-      <scope>compile</scope>
+      <groupId>net.sf.extjwnl</groupId>
+      <artifactId>extjwnl</artifactId>
+      <version>${extjwnl.version}</version>
+    </dependency>
+
+    <dependency>
+      <groupId>net.sf.extjwnl</groupId>
+      <artifactId>extjwnl-data-wn31</artifactId>
```

Comment:
This artifact has a rather special license (not BSD-3 as the main artifact). Can we add `<optional>true</optional>` here ?

Reply:
will try & update PR